assets/dev/scss/editor/panel/_elements.scss

```diff
@@ -96,8 +96,8 @@
 
 		.eicon-atomic {
 			position: absolute;
-			top: 5px;
-			@include end(5px);
+			inset-block-start: 5px;
```

Comment:
@rami-elementor Can you please check this for us?

Are we still supposed to used logical properties?

In this file, I am still seeing many cases of physical properties. I had expected that to be updated by now.

Reply:
@hein-obox This is correct.

Last week I merged the logical properties changes in atomic widgets.

Currently, I'm prioritising logical properties implement on frontend.

But you are correct, there are a lot of place in the Editor that we can change. It's in my TODO list.